build.gradle

```diff
@@ -25,7 +25,7 @@ ext {
 apply plugin: 'application'
 apply from: 'gradle/formatting.gradle'
 
-mainClassName = 'transportservice.ExtensionsRunner'
+mainClassName = 'opensearchSDK.transport.ExtensionsRunner'
```

Comment:
Can we pull out the runner from Transport? 
Conceptually Runner has nothing to do with Transport, it just initializes a bunch of data structures needed for an extension to run.